server_admin/topics/users/user-registration.adoc

```diff
@@ -1,30 +1,18 @@
-[[_user-registration]]
+[id="assembly-user-registration_{context}"]
+= User Registration
 
-=== User Registration
+[role="_abstract"]
+You can use {project_name} as a third-party authorization server to manage application users. Administrators can add users or users can register themselves.
 
-You can enable {project_name} to allow user self registration.  When enabled, the login page has a registration
-link the user can click on to create their new account.
+You can enable {project_name} to allow user self-registration. When enabled, the login page displays a registration link that the user can click to create an account.
 
-When user self registration is enabled it is possible to use the registration form to detect valid usernames and emails.
-It is also possible to enable <<_recaptcha,reCAPTCHA Support>>.
+When user self-registration is enabled, the user must complete a form to add profile information. 
 
-Enabling registration is pretty simple.  Go to the
-`Realm Settings` left menu and click it.  Then go to the `Login` tab.  There is a `User Registration` switch on this
-tab.  Turn it on, then click the `Save` button.
+The registration form can be customized by removing or adding additional fields that must be completed by a user. It is possible to use the registration form to detect a valid username and email.
 
-.Login Tab
-image:{project_images}/login-tab.png[]
+include::proc-enabling-user-registration.adoc[leveloffset=+2]
+include::proc-registering-new-user.adoc[leveloffset=+2]
 
-After you enable this setting, a `Register` link should show up on the login page.
-
-.Registration Link
-image:{project_images}/registration-link.png[]
-
-Clicking on this link will bring the user to the registration page where they have to enter in some user profile information
-and a new password.
-
-.Registration Form
-image:{project_images}/registration-form.png[]
-
-You can change the look and feel of the registration form as well as removing or adding additional fields that must be entered.
-See the link:{developerguide_link}[{developerguide_name}] for more information.
+[role="_additional-resources"]
+.Additional resources
+* For more information on configuring user registration, see the link:{developerguide_link}[{developerguide_name}].
```

Comment:
I think it should be "customizing" not "configuring" to tie into the previous sentences about "customizing."